<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VcsBase::Internal::CommonSettingsPage</class>
 <widget class="QWidget" name="VcsBase::Internal::CommonSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>180</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="lineWrapCheckBox">
     <property name="text">
      <string>Wrap submit message at:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="lineWrapSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> characters</string>
     </property>
     <property name="minimum">
      <number>40</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>72</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="submitMessageCheckScriptLabel">
     <property name="toolTip">
      <string>An executable which is called with the submit message in a temporary file as first argument. It should return with an exit != 0 and a message on standard error to indicate failure.</string>
     </property>
     <property name="text">
      <string>Submit message &amp;check script:</string>
     </property>
     <property name="buddy">
      <cstring>submitMessageCheckScriptChooser</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="Utils::PathChooser" name="submitMessageCheckScriptChooser" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="nickNameMailMapLabel">
     <property name="toolTip">
      <string>A file listing nicknames in a 4-column mailmap format:
'name &lt;email&gt; alias &lt;email&gt;'.</string>
     </property>
     <property name="text">
      <string>User/&amp;alias configuration file:</string>
     </property>
     <property name="buddy">
      <cstring>nickNameMailMapChooser</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Utils::PathChooser" name="nickNameMailMapChooser" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="nickNameFieldsFileLabel">
     <property name="toolTip">
      <string>A simple file containing lines with field names like &quot;Reviewed-By:&quot; which will be added below the submit editor.</string>
     </property>
     <property name="text">
      <string>User &amp;fields configuration file:</string>
     </property>
     <property name="buddy">
      <cstring>nickNameFieldsFileChooser</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="Utils::PathChooser" name="nickNameFieldsFileChooser" native="true"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="sshPromptLabel">
     <property name="toolTip">
      <string>Specifies a command that is executed to graphically prompt for a password,
should a repository require SSH-authentication (see documentation on SSH and the environment variable SSH_ASKPASS).</string>
     </property>
     <property name="text">
      <string>&amp;SSH prompt command:</string>
     </property>
     <property name="buddy">
      <cstring>sshPromptChooser</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="Utils::PathChooser" name="sshPromptChooser" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineWrapCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineWrapSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
